ecosystem/indexer-grpc/indexer-grpc-table-info/src/table_info_service.rs

```diff
@@ -295,8 +294,19 @@ impl TableInfoService {
         let end_version = raw_txns.last().unwrap().version;
         let num_transactions = raw_txns.len();
 
-        Self::parse_table_info(context.clone(), raw_txns.clone(), indexer_async_v2)
-            .expect("[Table Info] Failed to parse table info");
+        loop {
+            match Self::parse_table_info(
+                context.clone(),
+                raw_txns.clone(),
+                indexer_async_v2.clone(),
+            ) {
+                Ok(_) => break,
+                Err(e) => {
+                    error!(error = ?e, "Error during parse_table_info.");
```

Comment:
Would be helpful to add more context to the log or a comment here saying something like gRPC fails but rest of fullnode will continue